ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -25542,17 +25542,6 @@ periodics:
     job-release: "4.11"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-release-master-nightly-4.11-e2e-aws-single-node-workers
-  reporter_config:
-    slack:
-      channel: '#single-node-ci'
-      job_states_to_report:
-      - success
-      - failure
-      - error
-      report_template: '{{ if eq .Status.State "success" }} :green_jenkins_circle:
-        {{ else }} :red_jenkins_circle: {{ end }} Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
-        <{{.Status.URL}}|View logs> | <https://amd64.ocp.releases.ci.openshift.org/releasestream/4.11.0-0.nightly/release/{{index
-        .Annotations "release.openshift.io/tag"}}|Release status>'
```

Comment:
can you bring it back too?
it's not 100% managed by ``make update``, so that's probably why it's gone